config/routes.rb

```diff
@@ -134,7 +143,13 @@
   # yes, both of these are needed to override rails defaults of
   # /controller/:id/edit
   get '/app_configs/', to: 'app_configs#edit', as: :edit_app_configs # match
-  resources :app_configs, only: [:update]
+  resources :app_configs, only: %i() do
```

Comment:
this is also how we do this on [vesta](https://yale.githost.io/sdmp/rails/vesta/blob/master/config/routes.rb#L102). I know you had mentioned a better way to do this but I feel like this is cleaner than the `get 'photos/:id/:user_id', to: 'photos#show'` style solution